.github/workflows/release.yml

```diff
@@ -0,0 +1,43 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - "*"
+
+jobs:
+  release:
+    name: release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout commit and fetch tag history
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+
+      - name: Docker meta
+        id: docker_meta
+        uses: crazy-max/ghaction-docker-meta@v1
```

Comment:
This step looks a little over-engineered, but was used because it was the [recommended](https://github.com/docker/build-push-action#handle-tags-and-labels) approach to annotating the container image to be published .